README.md

```diff
@@ -31,10 +31,33 @@ docker --version
 Install these things to be able to run the application locally:
 
 #### Database
--- Insert guide here --
+This application uses PostgreSQL as database.
+To run the database locally, perform the following commands:
 
+If you haven't created the container before, create and start the container with the "docker run" command. Replace the "\*password\*" part with your own password. This must be the same password as the "DB_PASS" variable declared [here](https://github.com/mostafa-221/jobcrawler-backend/blob/4f238dfdbbcb624aa1bf2215282a9dcc7edd289e/src/main/resources/application.properties#L10).
 ```
-Give the example
+docker run --name jobcrawler-postgres -e POSTGRES_PASSWORD=*password* -d -p 5432:5432 postgres
+```
+
+If you did create the container before, run this:
```

Comment:
maybe "Once you have a container, run this to start it:"

Reply:
If you performed the docker run command, you don't have to use the start command, because the run command will also start it for you.
This sentence is a continuation after the "If you haven't created the container before, ..." on line 37.

Reply:
yes, you're correct. I didn't see the continuation

Reply:
I changed the sentence to make this a bit more clear